doc/rst/technotes/gpu.rst

```diff
@@ -0,0 +1,90 @@
+.. _readme-gpu:
+
+.. default-domain:: chpl
+
+=============================
+Support for GPU targeted code
+=============================
+
+Chapel includes preliminary work to target Nvidia GPUs using CUDA. This work is
+under active development and has not yet been tested under a wide variety of
+environments. We have tested it on systems with Nvidia Tesla P100 GPUs and CUDA
+11.0 and a system with Nvidia Tesla K20 GPUs with CUDA 10.2. The current
+implementation does not support multiple GPUs on one node and will only apply
+to certain `forall` loops.
+
+--------
```

Comment:
You don't need (shouldn't have?) `----`s above the inner headers.

Reply:
I'll take them out. Looks like some of our tech notes use 'em and others don't

Reply:
Fixed in 70f18f5727695ce8f553dd6920ac0714279336a3